content/about/_index.md

```diff
@@ -17,30 +17,37 @@ Who will be considered for CURIOSS membership? Individuals from universities or
     <p class="h1">Organizing Team</p>
   </div>
   <div class="col-xs-12 col-sm-6">
-    <img src="/images/about/team/ciara.png" title="Ciara Flanagan" class="align-middle"/>
+    <img style="height: 300px;" src="/images/about/team/ciara.png" title="Ciara Flanagan" class="align-middle"/>
```

Comment:
I believe this will cause bugs on some responsive views. What about putting the image in another div using the responsive classes?

Reply:
Thank you for the feedback! 
I updated it. Also needed to change the grid for your image section to make it more responsive.